validation.md

```diff
@@ -2241,6 +2244,192 @@ Occasionally, you may want to attach additional validation rules to your default
         // ...
     });
 
+<a name="validating-emails"></a>
+## Validating Emails
+
+Laravel provides a variety of validation rules that may be used to validate uploaded files, such as `rfc`, `strict` and `dns`. While you are free to specify these rules individually when validating emails, Laravel also offers a fluent email validation rule builder that you may find convenient:
+To ensure that emails are valid according to your application's requirements, you may use `Rule` class to fluently define the rule via ` Rule::email()`:
+
+```php
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+
+$validator = Validator::make($request->all(), [
+    'email' => ['required', Rule::email()],
+]);
+```
+The `Email` rule object allows you to easily customize how emails are validated for your application, such as specifying that emails require RFC compliance, DNS checks, or spoof detection:
+
+```php
+// Basic RFC compliance...
+Rule::email()->rfcCompliant();
+
+// Strict RFC compliance...
+Rule::email()->rfcCompliant(strict: true);
+
+// Check for valid MX records...
+Rule::email()->validateMxRecord();
+
+// Prevent spoofing...
+Rule::email()->preventSpoofing();
+```
+Of course, you may chain all the methods in the examples above:
+
+```php
+Rule::email()
+    ->rfcCompliant(strict: true)
+    ->validateMxRecord()
+    ->preventSpoofing();
+```
+
+> [!WARNING]  
+> The `validateMxRecord()` and `preventSpoofing()` validators require the PHP `intl` extension.
+
+<a name="defining-default-email-rules"></a>
+#### Defining Default Email Rules
+You may find it convenient to specify the default validation rules for emails in a single location of your application. You can easily accomplish this using the Email::defaults method, which accepts a closure. The closure given to the defaults method should return the default configuration of the `Email` rule. Typically, the `defaults` rule should be called within the boot method of one of your application's service providers:
+
+```php
+use Illuminate\Validation\Rules\Email;
+
+/**
+ * Bootstrap any application services.
+ */
+public function boot(): void
+{
+    Email::defaults(function () {
+        $rule = (new Email())->rfcCompliant(strict: true)->preventSpoofing(),
+
+        return $this->app->isProduction()
+                    ? $rule->validateMxRecord()
+                    : $rule;
+    });
+}
+```
+Then, when you would like to apply the default rules to a particular email undergoing validation, you may invoke the defaults method with no arguments:
+
+```php
+'email' => ['required', Email::defaults()],
+```
+Occasionally, you may want to attach additional validation rules to your default email validation rules. You may use the rules method to accomplish this:
+
+```php
+Email::defaults(function () {
+    return (new Email())->rfcCompliant(strict: true)
+      ->preventSpoofing()
+      ->rules(['ends_with:@example.com']);
+});
+```
+
+### Use Cases & Examples
```

Comment:
You might want to add a link up there, where the string rule is explained:
https://github.com/laravel/docs/pull/10112#issuecomment-2593840893